exhibitor/models.py

```diff
@@ -19,10 +19,36 @@ def __str__(self):
         return self.username
 
 class Moc(models.Model):
+    CASTLE = 'Castle/Starożytność'
+    ARCHITEKTURA = 'Architektura'
+    KOMIKS = 'Komiks/Anime/Manga'
+    MILITARIA = 'Militaria'
+    POJAZDY_MINIFIG = 'Pojazdy Minifig'
+    POJAZDY_TECHNIC = 'Pojazdy technic'
+    SPACE = 'Space/Sci-fi'
+    KOLEJNICTWO = 'Kolejnictwo'
+    CATEGORY = [
+        ('Castle/Starożytność', 'Castle/Starożytność'),
+        ('Architektura', 'Architektura'),
+        ('Komiks/Anime/Manga', 'Komiks/Anime/Manga'),
+        ('Militaria', 'Militaria'),
+        ('Pojazdy Minifig', 'Pojazdy Minifig'),
+        ('Pojazdy technic', 'Pojazdy technic'),
+        ('Space/Sci-fi', 'Space/Sci-fi'),
+        ('Kolejnictwo', 'Kolejnictwo'),
+    ]
+    FIRST_SIZE = '0,5 stołu'
+    SECOND_SIZE = '1 stół'
+    THIRD_SIZE = '2 stoły'
+    SIZE = [
+        ('0,5 stołu', '0,5 stołu'),
```

Comment:
from db perspective it would be more efficient if you do this field as float and then put choices like:

SIZE = [
         (0.5, '0,5 stołu'),
...]